components/style/properties/gecko.mako.rs

```diff
@@ -2776,6 +2776,34 @@ fn static_assert() {
     }
 </%self:impl_trait>
 
+<%self:impl_trait style_struct_name="InheritedBox"
+                  skip_longhands="image-orientation">
+    // FIXME: Gecko uses a tricky way to store computed value of image-orientation
+    //        within an u8. We could inline following glue codes by implementing all
+    //        those tricky parts for Servo as well. But, it's not done yet just for
+    //        convenience.
+    pub fn set_image_orientation(&mut self, v: longhands::image_orientation::computed_value::T) {
+        use properties::longhands::image_orientation::computed_value::T;
+        match v {
+            T::FromImage => {
+                unsafe {
+                    bindings::Gecko_SetImageOrientationAsFromImage(&mut self.gecko);
+                }
+            },
+            T::AngleWithFlipped(ref angle, flipped) => {
+                unsafe {
+                    bindings::Gecko_SetImageOrientation(&mut self.gecko, angle.radians() as f64, flipped);
+                }
+            }
+        }
+    }
+
+    pub fn copy_image_orientation_from(&mut self, other: &Self) {
```

Comment:
No idea what `Clone::clone_from` is...
Is this related to the clone_xxx_from()? I thought we need clone only for animatable properties, no?